scripts/release-stable.ts

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+import fs from 'fs-extra';
+import prompts from 'prompts';
+import sh from 'shelljs';
+
+import { resolveDir, changeAllPkgJSON, restorePkgJson } from './utils/changePackages';
+
+(async () => {
+  const mainPkgJSON = await fs.readJSON(resolveDir('./package.json'));
+
+  /**
+   * Change all package.json with the version from main package.json
+   * Run lerna to generate changelogs and bump version based on conventional commits
+   */
+  await changeAllPkgJSON(mainPkgJSON.version);
+  const args = `--conventional-commits --no-push --no-git-tag-version --no-commit-hooks --no-ignore-changes --yes`;
+  sh.exec(`pnpm lerna version ${args}`);
+
+  /**
+   * Restore dependencies versions and publish packages
+   */
+  await restorePkgJson(true);
+  const lernaJSON = await fs.readJSON(resolveDir('./lerna.json'));
+  const version = `v${lernaJSON.version}`;
+
+  const response = await prompts({
+    type: 'confirm',
+    name: 'confirm',
+    message: `Do you really want to bump to ${version}?`,
+  });
+
+  /**
+   * If confirm: publish using pnpm, Commit changed files and create a new tag
+   */
+  if (response.confirm) {
+    sh.exec(`pnpm publish -r --tag=latest --no-git-checks --force`);
+    sh.exec('git add ./CHANGELOG.md');
+    sh.exec('git add ./package.json');
+    sh.exec('git add ./lerna.json');
+    sh.exec('git add ./packages/**/CHANGELOG.md');
+    sh.exec('git add ./packages/**/package.json');
+    sh.exec(`git commit -m "bump to version ${version}"`);
+    sh.exec(`git tag -a ${version} -m "bump to version ${version}"`);
+    sh.exec('git push origin master --tags');
```

Comment:
I think it should create a PR

Reply:
Create a new branch with the version name and then push and open the PR. something like this.

Reply:
But if it will open a pull request, when will occur the `publish` execution for npm? 🧐 In fact, this command is just a small replacement of what @QuinnLee was already using with `lerna` commands globally, but now integrated with PNPM workspaces.

Because there's no sense create a pull request to be reviewed because if we're publishing a stable version, we already had reviews and just need to publish. This will be executed on `master` branch, just need a commit and tag in order to represent that at that point had a new version.